src/modules/common/Exchange.ts

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+
+    This file contain the Exchange class.
+
+    Copyright:
+        Copyright (c) 2021 BOSAGORA Foundation
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import bigDecimal from 'js-big-decimal';
+import { Amount } from 'boa-wallet-lib';
+import JSBI from 'jsbi';
+
+export class Exchange {
+    /**
+     * Current exchange of BOA/USD 
+     */
+    public exchangeRate: number
+
+    constructor(exchangeRate: number) {
+        this.exchangeRate = exchangeRate;
+    }
+
+    /**
+     * Convert BOA to USD.
+     * @param amount The amount.
+     * @returns Amount in USD.
+     */
+    public convertBoaToUsd(amount: number): number {
+        let usdAmount = bigDecimal.multiply(this.exchangeRate, amount);
+        return Number(usdAmount);
+    }
+
+    /**
+     * Convert Amount To USD.
+     * @param amount The amount.
+     * @returns Amount in USD.
+     */
+    public convertBalanceToUsd(amount: JSBI) {
+        return this.convertBoaToUsd(Number(Amount.toString(amount)));
```

Comment:
https://github.com/bosagora/boa-sdk-ts/blob/ce93bd29169e98ee9a55800f6f5729eccfa9dab9/src/modules/common/Amount.ts#L298
You can use this.